error.go

```diff
@@ -24,7 +27,7 @@ type DefaultErrors struct {
 	errormap map[string]string
 }
 
-// NewDefaultErrors initializes OAuth2 error codes and descriptions.
+// NewDefaultErrors initializes OAuth2 & Accelbyte extensions error codes and descriptions.
```

Comment:
I think we do  not need treat it as Accelbyte's special error. This is a common OAuth error.

Reply:
I think it should belongs to Extension errors, in osin design, this error file only includes standard error set